### PR TITLE
zehra_week1

### PR DESCRIPTION
**Pull request by zehrakopar**
1 -index.html
2-app.js
3-style.css

### CODE REVIEW

File: homework-submission/week1/app.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  render() {
+    return (
+      <div className="dynamic_list">
+        <h1>Dynamic List</h1>
+        <ul>
+          {dynamicListArray.map(listItem => {
+            return (
+              <TodoListItem
+                key={listItem.id}
+                className={listItem.done === true ? "done" : "not_done"}
```
Comment (ArcoMul) [inline homework-submission/week1/app.js]: In my opinion it isn't so pretty to pass className as prop. It's more clear to decide in the `TodoListItem` component which className to add based on a prop called `done`.

File: homework-submission/week1/style.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+.dynamic_list {
+  background-color: rgb(17, 97, 124);
+  color :white;
+  margin: 10px;
+  padding:10px;
+}
+
+.done {
+  text-decoration: underline;
```
Comment (ArcoMul) [inline homework-submission/week1/style.css]: `line-through` is more clear that a todo item is done

File: homework-submission/week1/app.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
+// Create a todo list item as reusable component:
```
Comment (ArcoMul) [inline homework-submission/week1/app.js]: The description of this component (and which props it accepts) could be a bit more elaborate in my opinion

File: homework-submission/week1/app.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  },
+];
+// Create a todo list called Dynamic List with the items of the dynamicListArray
+class DynamicList extends React.Component {
+  render() {
+    return (
+      <div className="dynamic_list">
+        <h1>Dynamic List</h1>
+        <ul>
+          {dynamicListArray.map(listItem => {
```
Comment (ArcoMul) [inline homework-submission/week1/app.js]: You're using a global variable in your component. In this case you can easily pass the array as a prop to the component.

File: homework-submission/week1/app.js (hunk 1 above)
Comment (zehrakopar) [reply homework-submission/week1/app.js]: i ve got it..

File: homework-submission/week1/style.css (hunk 2 above)
Comment (zehrakopar) [reply homework-submission/week1/style.css]: Certainly..

File: homework-submission/week1/app.js (hunk 3 above)
Comment (zehrakopar) [reply homework-submission/week1/app.js]: I wrote a detailed comment  in the changed file(2. comment)

File: homework-submission/week1/app.js (hunk 4 above)
Comment (zehrakopar) [reply homework-submission/week1/app.js]: Done in the changed file(2. commit)

File: homework-submission/week1/app.js (hunk 3 above)
Comment (zehrakopar) [reply homework-submission/week1/app.js]: (2.commit)

**Comment by ArcoMul** [comment]
Great work @zehrakopar! 🙌

**Pull request closed without merging**